### PR TITLE
app-shells/tasksh: Added tasksh 1.2.0

### PR DESCRIPTION
**Pull request by arzano**
Tasksh is a shell for Taskwarrior.

Bug: https://bugs.gentoo.org/673534
Package-Manager: Portage-2.3.69, Repoman-2.3.11

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
Hi there,

Sometimes 'new package' PRs get snowed under by all the other ones (new packages are low priority compared to fixes to already existing packages). If you'd like to make this ebuild available to the Gentoo users anyway, might I suggest the [GURU overlay](https://wiki.gentoo.org/wiki/Project:GURU). This is an ebuild repository maintained collaboratively by Gentoo users. To gain access to the repository, simply create an "Access request" on bugs.gentoo.org.

File: app-shells/tasksh/metadata.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+
+<pkgmetadata>
+  <maintainer type="person">
+    <email>max@magorsch.de</email>
```
Comment (ajakk) [inline app-shells/tasksh/metadata.xml:6]: A different email might be more appropriate now ;)

File: app-shells/tasksh/tasksh-1.2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (ajakk) [inline app-shells/tasksh/tasksh-1.2.0.ebuild:4]: Can we upgrade to EAPI=7?

File: app-shells/tasksh/tasksh-1.2.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+inherit cmake-utils
```
Comment (ajakk) [inline app-shells/tasksh/tasksh-1.2.0.ebuild:6]: `cmake-utils` has been deprecated, can we port to cmake.eclass?

File: app-shells/tasksh/tasksh-1.2.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+
+DEPEND="app-misc/task
+	sys-libs/readline:0"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	      cmake-utils_src_prepare
```
Comment (ajakk) [inline app-shells/tasksh/tasksh-1.2.0.ebuild:21]: Looks like a tab and a bunch of spaces here, and the below indented blocks are multiple tabs, let's be consistent with one tab per indentation?

**Review by ajakk — CHANGES_REQUESTED** [review]

**Comment by ajakk** [comment]
Feel free to reopen if you continue working on this.

**Pull request closed without merging**